Browser_IDE/executionEnvironment_Internal.js

```diff
@@ -515,4 +524,29 @@ moduleEvents.addEventListener("onRuntimeInitialized", function() {
     });
 
     parent.postMessage({type:"initialized"},"*");
-});
\ No newline at end of file
+});
+
+function handleUserInput(event) {
+    if (event.key === "Enter") {
+        console.log("Event triggered:", event);
+        event.preventDefault();
+        const input = document.getElementById('user-input');
+        const output = document.getElementById('output');
+        const command = input.value.trim();
+        console.log("User input:", command);
+        input.value = ''; // Clear the input field
+
+        if (window.userInputResolve) {
+            console.log("Output element before update:", output.innerHTML);
+            output.innerHTML += `&gt; ${command}<br>`; // Use &gt; to represent '>' in HTML
+            console.log("Output element after update:", output.innerHTML);
+            console.log("Resolving user input with command:", command);
+            window.userInputResolve(command); // Resolve the Promise with the command
+            window.userInputResolve = null; // Reset the resolve function
+        } else {
+            console.log("No userInputResolve function available");
+        }
```

Comment:
Just replace this bit with:
```suggestion
        writeTerminal("> " + command);

        if (window.userInputResolve) {
            window.userInputResolve(command); // Resolve the Promise with the command
            window.userInputResolve = null; // Reset the resolve function
        }
```